src/pages/Features/Features.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useEffect, useState } from 'react'
+import ReactDOM from 'react-dom'
+import { message } from 'antd'
+import { useMoralis } from 'react-moralis'
+import { FeatureCard } from './FeatureCard'
+import ModalComponent from './submitFeature'
+
+const Features = () => {
+  const { Moralis, isInitialized, account, isAuthenticated } = useMoralis()
+
+  async function likeFeature(id) {
+    /* like a feature */
+
+    const FeatureObject = await Moralis.Object.extend('Feature')
+    const query = new Moralis.Query(FeatureObject)
+    query.equalTo('objectId', id)
+    const result = await query.find()
+    let response = { likes: result[0].attributes.likes, isLiked: false }
```

Comment:
It would be better to save the first element to a separate variable rather than always accessing the first array element